GliaWidgets/Public/Glia/Glia.OpaqueAuthentication.swift

```diff
@@ -38,6 +38,24 @@ extension Glia.Authentication.Behavior {
 }
 
 extension Glia {
+    /// Initiates authentication with specified behavior, manages UI state reset, 
+    /// and handles authentication operations.
+    ///
+    /// - Parameters:
+    ///   - behavior: The behavior defining how authentication should be handled.
+    ///
+    /// - Throws:
+    ///   - `GliaCoreError(reason: "SDK is not configured properly. Make sure it is fully configured and try again.")`
```

Comment:
Is this the only possible error with such message possible?

Reply:
It appears so. I looked through CoreSDK and that's the only error that was thrown there with a hard coded reason string value